### PR TITLE
Create vela.json

### PR DESCRIPTION
**Pull request by v3laDev**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x ] The file is in the `domains` folder and is in the JSON format.
- [ x] The file's name is all lowercased and alphanumeric. 
- [x ] You have completed your website. 
- [ x] The website is reachable.  
- [ x] The CNAME record doesn't contain `https://` or `/`.  
- [ x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/vela.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "v3laDev",
+        "email": "27khero@proton.me"
+    },
+    "record": {
+        "RECORDTYPE": "CNAME"
```
Comment (DEV-DIBSTER) [inline domains/vela.json:7]: Your RECORDTYPE should be an actual record and the value should be the value you want to place for that record.

See docs: https://docs.is-a.dev

**Review by DEV-DIBSTER — DISMISSED** [review]

File: domains/vela.json (hunk 1 above)
Comment (v3laDev) [reply domains/vela.json]: alr

File: domains/vela.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "v3laDev",
+        "email": "27khero@proton.me"
+    },
+    "record": {
+        "RECORDTYPE": "v3laDev.github.io"
```
Comment (v3laDev) [inline domains/vela.json:7]: changed

**Review by v3laDev — COMMENTED** [review]
changed recordtype

**Review by wdhdev — APPROVED** [review]

**Comment by v3laDev** [comment]
ty

**Pull request merged**